crates/bevy_window/src/resolution.rs

```diff
@@ -0,0 +1,101 @@
+//! Provides [`PhysicalSize`] and [`LogicalSize`] structs to help resolution calulations.
+
+use std::ops::Div;
+
+use bevy_math::{UVec2, Vec2};
+use bevy_reflect::Reflect;
+
+/// A [`PhysicalSize`] is the width and height using physical pixels.
+#[derive(Reflect, Default, Debug, Copy, Clone, PartialEq)]
+pub struct PhysicalSize {
```

Comment:
If the fields are going to be called `x` and `y` it probably makes more sense to make this (and logical size) a newtype wrapping `UVec2` and deriving `Deref` and `DerefMut`.

Reply:
I didn't think about wrapping `UVec2` into these types! That does make more sense as I think that would make operations with other U/Vec2 easier/cleaner, right? Are you also suggesting to make `LogicalSize` wrap a `UVec2`? From what I saw, everything in logical units was a f32 so wouldn't it make more sense to wrap a `Vec2`?